apps/visualization/rest_filters.py

```diff
@@ -19,3 +24,31 @@ def filter_topic(self, qs, name, value):
             return qs
         topics = [topic.lstrip() for topic in value.split(',')]
         return qs.filter(topic__in=topics)
+
+
+class BaseExportFilter(filters.FilterSet):
+    iso3 = filters.CharFilter(field_name='iso3')
+    indicator_id = filters.CharFilter(field_name='indicator_id')
+    include_header = filters.BooleanFilter(method='filter_include_header', required=True)
+
+    def filter_include_header(self, queryset, name, value):
+        # Used for browseable api documentation
+        return queryset
+
+
+class DataCountryLevelPublicExportFilter(BaseExportFilter):
+    class Meta:
+        model = DataCountryLevelPublic
+        fields = ()
+
+
+class DataGranularPublicExportFilter(BaseExportFilter):
+    class Meta:
+        model = DataGranularPublic
+        fields = ()
+
+
+class ContextualDataExportFilter(BaseExportFilter):
+    class Meta:
+        model = ContextualData
+        fields = ()
```

Comment:
Maybe we can remove this as they are all same for now.

Reply:
@thenav56  This is changed now,  ContextualData does't have `indicator_id` field we have to filter by 'context_indicator_id'

Reply:
@bimal125